README.md

```diff
@@ -1,50 +1,38 @@
-
-#### Pulumi Steps
+    * `docs/_index.md`, which should contain a summary of the provider's purpose (required) along with a code sample in each language (required). This file will render as the index page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/)).
+    * `docs/installation-configuration.md`, which should contain links to published SDKs in (Go, Typescript, C#, Python; required) along with instructions for configuring the provider (e.g., required credentials and/or environment variables). This file will render as the Installation & Configuration page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/installation-configuration/)).
+1. Create a release of your provider in GitHub with a "v" + [Semver 2.0](https://semver.org) compliant tag (`vX.Y.Z`).
 
-Once the community member has submitted the PR to add the provider to the registry, a Pulumi staff member should perform the following steps:
+## Internal Process
 
-1. Review the PR. Ensure that the PR has been rebased if necessary before merging. If ok, merge.
-1. Once the PR is merged, a [scheduled task](https://github.com/pulumi/registry/actions/workflows/generate-package-metadata.yml) will pick up the changes and create a PR to add the package metadata to the registry. A correct metadata PR ([example](https://github.com/pulumi/registry/pull/1606/files)) will include the following files, at a minimum:
-   * `data/registry/${PROVIDER}.yaml` which includes structured metadata about the provider. This file is always included with every PR that gets generated for a new release.
-   * `themes/default/content/registry/packages/${PROVIDER}/installation-configuration.md`, as described above. This file *must* be included in the first release, but will only be included in subsequent PRs if the content has changed.
-   * `themes/default/content/registry/packages/exoscale/_index.md`, as described above. This file *must* be included in the first release, but will only be included in subsequent PRs if the content has changed.
+When a community member requests adding a provider to the registry, a Pulumi staff member should perform the following steps:
 
-   Optionally, the PR may include additional content files like How-To Guides if they are present in the upstream repo.
-
-1. Merge the PR if it looks ok.
-1. In pulumi/docs, a [scheduled task](https://github.com/pulumi/docs/actions/workflows/update-theme.yml) runs hourly and will pick up any changes in this repo, generate files from the provider schema and `data/registry/${PROVIDER}.yaml`, and publish to pulumi.com.
-
-  This scheduled task currently lacks adequate monitoring, and **should be watched to ensure that it runs correctly to completion**. (If it fails, it will block all updates to pulumi.com, including marketing and manually maintained docs pages.)
+1. Add the package to the [community packages list](./community-packages/package-list.json) via pull request to this repository.
```

Comment:
The numeration isn't correct here (seems like it was also wrong before). At least they render correctly because markdown doesn't really care

Reply:
This is deliberate - all numbers are 1 so that diffs aren't bad if one extra point is added later on causing all the numbers to shift.

Reply:
Fair enough!